index.bs

```diff
@@ -221,6 +221,23 @@ and doing cleanup as necessary (see the [[#listen-fully-active]] pattern above).
   but every feature specification should think about this and decide what works best for the feature.
 </div>
 
+<h4 id="non-synchronous-steps">Be aware that navigations can happen between steps running [=in parallel=]</h4>
+While running steps [=in parallel=],
+the document the steps are running on or are otherwise using might be navigated away and navigated to again.
+The steps themselves will not run while the document is not [=fully active=],
```

Comment:
Actually, they will. There's nothing stopping in parallel steps at the moment. What won't run is steps in a queued task if that task is associated with the document.

Reply:
Ah, right. I've rewritten this to mention that possibility, and add a suggestion to maybe not run some steps while the document is fully active. PTAL again!